includes/multi-currency/class-currency.php

```diff
@@ -157,6 +171,15 @@ public function get_symbol() {
 		return get_woocommerce_currency_symbol( $this->code );
 	}
 
+	/**
+	 * Retrieves if the currency is zero decimal.
+	 *
+	 * @return bool
+	 */
+	public function get_is_zero_decimal() {
```

Comment:
Nitpick: Would you mind adding type hinting to the function declaration here? (e.g. `public function get_is_zero_decimal(): bool {`)

Reply:
You're right! I missed that looking at the others methods. I added types for every method in f288f19.

Reply:
Thanks a lot, @ismaeldcom! That's great! 👍